### PR TITLE
Release of the XWiki 12.6.2 special branch

### PR DESCRIPTION
**Pull request by vmassol**

### CODE REVIEW

File: library/xwiki [hunk 1]
```diff
@@ -3,23 +3,34 @@
...
+
+# This tag description should be removed once the XWiki 12.x LTS is out
+Tags: 12.6, 12.6.2, 12.6.2-mysql-tomcat
+Architectures: amd64
+Directory: 12/mysql-tomcat
+GitCommit: 9fdf9a8c0ec050c74f6a5f10578fa6a900e5decd
+GitFetch: refs/heads/stable-2.6.x
+
+# This tag description should be removed once the XWiki 12.x LTS is out
+Tags: 12.6.2-postgres-tomcat
```
Comment (surli) [inline library/xwiki:32]: missing 12.6-postgres-tomcat tag

File: library/xwiki [hunk 2]
```diff
@@ -3,23 +3,34 @@
...
-GitCommit: 951343bef4867540886b559a8c1fece56a1b6fd3
 Directory: 12/mysql-tomcat
 
 Tags: 12-postgres-tomcat, 12.7-postgres-tomcat, 12.7.1-postgres-tomcat, postgres-tomcat, stable-postgres-tomcat, stable-postgres
 Architectures: amd64, arm64v8
-GitCommit: 951343bef4867540886b559a8c1fece56a1b6fd3
 Directory: 12/postgres-tomcat
+
+# This tag description should be removed once the XWiki 12.x LTS is out
+Tags: 12.6, 12.6.2, 12.6.2-mysql-tomcat
```
Comment (surli) [inline library/xwiki:25]: missing 12.6-mysql-tomcat tag.

File: library/xwiki (hunk 1 above)
Comment (vmassol) [reply library/xwiki:35]: Fixed thanks

File: library/xwiki (hunk 2 above)
Comment (vmassol) [reply library/xwiki]: Fixed too, thanks

**Pull request merged**